locale/en/get-involved/node-speakers.md

```diff
@@ -17,6 +17,20 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 
 ## Speakers
 
+### Brazil 🇧🇷
+
+#### São Paulo
+
+##### São Paulo 
```

Comment:
Are you sure this duplication is OK with 'São Paulo'?

Reply:
IIUC, the first means `state`, and second means `city`.

Refs: https://github.com/nodejs/nodejs.org/pull/2005#pullrequestreview-196402720

Reply:
@ZYSzys：So 'São Paulo' is a state and a city?

Reply:
Maybe yes.
![screen shot 2019-02-10 at 12 24 43 pm](https://user-images.githubusercontent.com/23313266/52529582-e548a680-2d2e-11e9-9e61-bccd11a43280.png)

Refs: https://en.wikipedia.org/wiki/S%C3%A3o_Paulo

Reply:
OK